app/(app)/(purchase-and-payment)/purchase/choose-vehicle.tsx

```diff
@@ -89,9 +89,7 @@ const ChooseVehicleScreen = () => {
         </View>
 
         <View className="flex-1 g-2">
-          <Typography variant="default-bold" className="grow">
```

Comment:
This `grow` definitely does something when used with Typography. I remember that we added in on purpose. But I don't remember, and I don't know if also in this place, but it helps to handle long words/texts I guess?